src/ape/pytest/plugin.py

```diff
@@ -25,6 +25,11 @@ def pytest_addoption(parser):
         action="store_true",
         help="Open an interactive console each time a test fails",
     )
+    parser.addoption(
```

Comment:
Would it be challenging to add a test in integration/cli/test_test.py to make sure when this option is passed that things are not isolated?

Reply:
ahhh definitely need to add that, :pray: